CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed spurious info output in terminal by moving all list patterns under `molecules/lists`
 - Removed unnecessary universal selector inside hero component, which caused layout issues in IE
 - Fixed SVGs rendering at the wrong size in IE
+- Added a max width to the logo so there's more space for the primary nav
+- Primary nav will now wrap underneath the logo if there's not enough space for it
+
 
 ### Changed
 - Increased size and spacing of burger menu strokes
```

Comment:
Please add a note under "Changed" about renaming `grav-c-block-link` to `grav-c-nav-link` and call out that it's a _breaking change_.